docs/operate/reference/components/servo/_index.md

```diff
@@ -25,6 +25,8 @@ If you have a physical ["RC" or "hobby" servo motor](https://learn.adafruit.com/
 
 If your motor is coupled with an [encoder](/operate/reference/components/encoder/), not a potentiometer, for position feedback, you should configure it as an [encoded motor](/operate/reference/components/motor/encoded-motor/) instead.
 
+If you have an industrial servo motor, configure a [motor component](/operate/reference/components/motor/) that supports your hardware.
+
```

Comment:
I would suggest recommending that they configure an appropriate motor model, or recommend make a module of their own and link to the module page.

Reply:
```suggestion
If you have an industrial servo motor, configure or [create](/operate/get-started/other-hardware/) a [motor component](/operate/reference/components/motor/) that supports your hardware.

```